src/Nethermind/Nethermind.Consensus/Processing/BlockchainProcessor.cs

```diff
@@ -439,6 +441,11 @@ public bool IsProcessingBlocks(ulong? maxProcessingInterval)
         return maxProcessingInterval is null || _lastProcessedBlock.AddSeconds(maxProcessingInterval.Value) > DateTime.UtcNow;
     }
 
+    private bool IsNethermindBlock(Block block)
+    {
+        return Encoding.ASCII.GetString(block.ExtraData).Contains(BlocksConfig.DefaultExtraData, StringComparison.InvariantCultureIgnoreCase);
```

Comment:
Make sure that it won't throw any exception by unit test or sth. If this method is going to throw something we have a problem. ExtraData can be null in theory